plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -488,6 +579,8 @@ def main():
         confirm=dict(type='bool', default=True),
         isadmin=dict(type='bool', default=False),
         external=dict(type='bool', default=False),
+        identities=dict(type='list'),
```

Comment:
```suggestion
        identities=dict(type='list', elements='dict', options=dict(...)),
```
(Need to fill in the spec for the elements.)

Reply:
should be fixed